### PR TITLE
Added paragraph providing more details on QubesOS network to prepare for update on firewall config

### PR DESCRIPTION
**Pull request by adubois**

### CODE REVIEW

File: security/firewall.md [hunk 1]
```diff
@@ -99,6 +99,34 @@ default Qubes installation):
...
+- The DMZ: the topologies does not change from the default tree, just some of the ApplicationVM host a service (i.e. print/file/web server)
+
+The network interface connecting to the NetworkVM of a qube is eth0 while interface(s) connecting to the leaves are vifXX.0.
+In R4: DisposableVM have an IP address of 10.138.XX.YY/32 while the other VMs have a fixed IP address of 10.137.0.XX/32 (vifXX.0 are not fixed).
+For a good visibility, from Dom0 terminal
+~~~
+qvm-ls -n
+~~~
+
+In order to provide more security and anonymity, in QubesOS all the virtual MAC addresses are fe:ff:ff:ff:ff:ff. Moreover, routing is kept local between a qube and its NetworkVM (no route is present in any qube). This is achieved by each qube having his NetworkVM as default gateway while natting (masquerading) the traffic (hiding traffic coming from qube's child behind it's own IP).
```
Comment (marmarek) [inline security/firewall.md]: The fe:ff:ff:ff:ff:ff mac is on the backend (vif) side. The frontend (eth0) side have 00:16:3e:5e:6c:00 address. And in R4.0 can be changed per VM (`mac` property).

**Comment by adubois** [comment]
@andrewdavidwong is there a common terminology for relations between VMs. I instinctively see the AppVM as upstream and NetVM as downstream (which has a similar representation with trees, root and leaves).

**Comment by unman** [comment]
@adubois
If it makes any sense in this context at all,(and I'm not sure it does), it would usually be the other way round.
In comms/network usually downstream means movement from service provider to customer: in server/client model it means the direction from server to client. So traffic from (e.g) website to browser in a qube could be described as downstream.
By analogy traffic from sys-net to firewall to client _could_ be described as moving downstream.
To me it makes sense to show this:
sys-net -> firewall -> qube
But better might be sys-net <--> firewall <--> qube

**Comment by adubois** [comment]
@marmarek, I've done few diagrams. How do I incorporate them, I looked at the Split-gpg page, the markdown refer to /attachment/wiki/SplitGpg/diagram.png.

**Comment by adubois** [comment]
@unman OK, no problem. I don't mind as long as it is the usual terminology used within the Qubes community.
So unless you are not sure that it is or someone comes along, I'll go with your suggestion.

**Comment by tasket** [comment]
FYI, downstream = AppVM is how the VPN doc terms it.

**Comment by adubois** [comment]
I think I am going to create a new pull request. Github, instead of creating a new branch renamed the branch I was on when I committed some other changes. My fault anyway. So unless you can access only commits up to "formating fix"... I'll just create a new branch and put the changes there... Sorry to be a noob.

**Comment by marmarek** [comment]
> So unless you can access only commits up to "formating fix"...

Yes, that's easy. Unless some more changes will be needed (additional commits).

**Comment by andrewdavidwong** [comment]
> Yes, that's easy. Unless some more changes will be needed (additional commits).

Given that additional commits are required, it sounds like it might be a good idea to open a new PR.

**Comment by adubois** [comment]
Erroneous merge. Will re-submit a new pull request.

**Pull request closed without merging**